src/App/App.js

```diff
@@ -47,31 +60,43 @@ class App extends Component {
   }
   
   render() {
-    return (
-      <div className="App">
-        <Header 
-          className="header"
-          searchMovies={this.searchMovies}
-          handleChange={this.handleChange}
-          searchTerm={this.state.searchTerm}
-        />
-
-        <Route exact path="/" render={() => 
-          <CardContainer
-            className="card-container" 
-            movieData={!this.searchMovies(this.state.searchTerm).length ? 
-              this.state.movieData : this.searchMovies(this.state.searchTerm)} 
+    if (this.state.errorThrown) {
```

Comment:
Now we are setting state when the data comes in, and the render function is looking at state to decide what to render.